src/mjolnir/valhalla_add_landmarks.cc

```diff
@@ -51,7 +49,7 @@ int main(int argc, char** argv) {
     valhalla::midgard::logging::Configure(logging_config);
   }
 
-  if (!valhalla::mjolnir::AddLandmarks(pt.get_child("mjolnir"))) {
+  if (!valhalla::mjolnir::AddLandmarks(pt)) {
```

Comment:
```suggestion
  if (!valhalla::mjolnir::AddLandmarks(pt.get_child("mjolnir"))) {
```
I've edited AddLandmarks to handle the input as child mjolnir, maybe we want to change this back to get child mjolnir. it works for me :)

Reply:
Yeah I must not've seen that for some reason. Weird though that I could push at all if I didn't pull your changes before.. I'll revert this!

Reply:
I guess you can push because there is no conflict caused by this commit